Backend/src/sqlite.py

```diff
@@ -6,85 +6,216 @@ def connect_to_db():
     return conn
 
 
-def create_db_table():
+# Rooms
+
+def create_table_room():
+    try:
+        conn = connect_to_db()
+        conn.execute('''
+            CREATE TABLE Rooms (
+                id INTEGER PRIMARY KEY NOT NULL,
+                name TEXT UNIQUE NOT NULL,
+                state TEXT
+            );
+        ''')
+
+        conn.commit()
+        print("rooms table created successfully")
+    except:
+        print("rooms table creation failed - Maybe table")
+    finally:
+        conn.close()
+
+
+def add_room(room):
+    inserted_room = {}
+    try:
+        conn = connect_to_db()
+        cur = conn.cursor()
+        cur.execute("INSERT INTO rooms (name) VALUES ('{}')".format(room))
```

Comment:
First off, I have no idea how your framework works - so please excuse me if this is dumb;
if I call `add_room('my room \'); DROP TABLE *;(\'')`, wouldn't this exploit the DB?